grails-app/controllers/de/iteratec/osm/result/EventResultDashboardController.groovy

```diff
@@ -159,6 +159,10 @@ class EventResultDashboardController {
         }
 
         TimeSeriesChartDTO timeSeriesChartDTO = lineChartTimeSeriesService.getTimeSeriesFor(cmd)
+        if (!timeSeriesChartDTO.any { it.series.size() != 0 }) {
```

Comment:
I think this is easier to read:
```suggestion
        if (timeSeriesChartDTO.each { it.series.size() == 0 }) {
```

Reply:
I think that `.every` would be better suited here, but otherwise I agree and will change that.